src/ast/mod.rs

```diff
@@ -464,6 +464,7 @@ pub enum Statement {
     CreateView {
         /// View name
         name: ObjectName,
+        or_replace: bool,
```

Comment:
Why did you put `or_replace` between `name` and `columns`?

Reply:
I would be fine with any order, as long as it can be explained:)

What do you think about using the source code order, when possible? I.e. `or_replace`, `materialized`, `name`, ...?

Reply:
Makes sense, also applied that for CreateTable:
```
        or_replace
        external
        if_not_exists
        name
```